core/lib/zksync_core/src/lib.rs

```diff
@@ -1,4 +1,8 @@
-#![allow(clippy::upper_case_acronyms, clippy::derive_partial_eq_without_eq)]
+#![allow(
+    clippy::upper_case_acronyms,
+    clippy::derive_partial_eq_without_eq,
+    clippy::redundant_locals
```

Comment:
Bikeshedding: IMO, more local placement of this switch (e.g., on the `consensus` module, or even on specific methods / functions) would be better.

Reply:
done

Reply:
done